bucketeer/src/main/kotlin/io/bucketeer/sdk/android/internal/event/db/EventDaoImpl.kt

```diff
@@ -41,6 +43,7 @@ internal class EventDaoImpl(
     return c.use {
       c.asSequence()
         .mapNotNull { eventAdapter.fromJson(it.getString(COLUMN_EVENT)) }
+        .map { it.copy(event = updateSdkVersion(it.event)) }
```

Comment:
I'm currently sending the current sdk_version, but should we send a sdk_version of when it tracked actually?

Reply:
I think it will be more accurate if we send it using the version when the event was created, not when it was sent.
For existing events when the SDK is updated, I think we could leave it as it is (empty).
What do you think?

Reply:
That sounds reasonable.
Thanks, I'm going to rework it.